### ISSUE TITLE
Update `libiec61850` to 1.6

### ISSUE THREAD

**Issue by pjungkamp** [VILLASframework/node#819]
`libiec61850` has been updated to 1.6.0

This release includes experimental support for IEC61850-90-5 Routed-GOOSE and Routed-SV, but also breaks API.

We probably don't need to keep support for the 1.5 releases, since we're only using a packaged version of `libiec61850` on NixOS. Other distros depend on `deps.sh`.

This revision has entered `nixpkgs-unstable` and as such is blocking `nix flake update`.

**Comment by stv0g** [comment]
I agree. Lets switch to 1.6 and drop backward compatability with pre 1.6 versions.

**Comment by stv0g** [comment]
Done
